scanner/templates/node/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:bullseye as builder
+
+ARG NODE_VERSION={{ .nodeVersion }}
+{{ if .yarn -}}
+ARG YARN_VERSION={{ .yarnVersion }}
+{{ end }}
+RUN apt-get update; apt install -y curl
+RUN curl https://get.volta.sh | bash
+ENV VOLTA_HOME /root/.volta
+ENV PATH /root/.volta/bin:$PATH
+{{ if .yarn -}}
+RUN volta install node@${NODE_VERSION} yarn@${YARN_VERSION}
+{{ else -}}
+RUN volta install node@${NODE_VERSION}
+{{ end }}
+#######################################################################
+
+RUN mkdir /app
+WORKDIR /app
+
+ENV NODE_ENV production
```

Comment:
Would we also want node-env set to production in the runner definition below?

Reply:
good catch!